examples/exotica_examples/scripts/example_ik_lookat.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+import rospy
+import pyexotica as exo
+import numpy as np
+import signal
+from pyexotica.publish_trajectory import publishPose, sigIntHandler
+from time import sleep
+import task_map_py
+from sensor_msgs.msg import Joy, JointState
+from std_msgs.msg import Float64MultiArray as FloatArray
+from visualization_msgs.msg import Marker
+from std_msgs.msg import ColorRGBA as Color
+
+GREEN = Color()
+GREEN.r=0.0
```

Comment:
PEP8 formatting

Reply:
Do you mean the constant variable `GREEN`? If so, it does follow PEP8, see [here](https://www.python.org/dev/peps/pep-0008/#constants).

Reply:
I meant the spacing around `=`

Reply:
Ahh I see, agreed